tools/config/go-templates/model_anyof.mustache

```diff
@@ -1,3 +1,5 @@
+// THIS TEMPLATE IS CURRENTLY NOT BEING USED
```

Comment:
FYI: To be precise they are used by openapi generator.
Current model do not contain data that trigger output for template.

Reply:
thanks, adjusting message to:  THIS TEMPLATE IS NOT BEING TRIGGERED BY THE CURRENT MODEL

Reply:
SGTM